docs/api.md

```diff
@@ -327,7 +328,35 @@ will be closed.
 If no URLs are specified, this method returns all cookies.
 If URLs are specified, only cookies that affect those URLs are returned.
 
-> **NOTE** the default browser context cannot be closed.
+#### browserContext.evaluateOnNewDocument(pageFunction[, ...args])
+- `pageFunction` <[function]|[string]> Function to be evaluated in all pages in the browser context
+- `...args` <...[Serializable]> Arguments to pass to `pageFunction`
+- returns: <[Promise]>
+
+Adds a function which would be invoked in one of the following scenarios:
+- whenever the page is navigated
+- whenever the child frame is attached or navigated. In this case, the function is invoked in the context of the newly attached frame
```

Comment:
whenever a child frame is attached or navigated in a page in the browser context